src/Codeception/Module/Doctrine2.php

```diff
@@ -397,6 +397,37 @@ public function grabFromRepository($entity, $field, $params = [])
     }
 
     /**
+     * Selects entities from repository.
+     * It builds query based on array of parameters.
+     * You can use entity associations to build complex queries.
+     *
+     * Example:
+     *
+     * ``` php
+     * <?php
+     * $users = $I->grabEntitiesFromRepository('User', array('name' => 'davert'));
+     * ?>
+     * ```
+     *
+     * @version 1.1
+     * @param $entity
+     * @param array $params
+     * @return array
+     */
+    public function grabEntitiesFromRepository($entity, $params = [])
```

Comment:
Thanks. Could you also add `grabEntityFromRepository` to get a single entity or fail?